benchmark/Streamly/Benchmark/Prelude/ZipAsync.hs

```diff
@@ -41,6 +41,11 @@ zipAsyncAp count n =
     S.fromZipAsync $
         (,) <$> sourceUnfoldrM count n <*> sourceUnfoldrM count (n + 1)
 
+fromZipAsyncTraverse100 :: Benchmark
+fromZipAsyncTraverse100 =
+    bench "ZipAsync Applicative"
+        $ nfIO $ S.drain $ S.fromZipAsync $ traverse S.fromPure [0 :: Int .. 99]
```

Comment:
Why 100? For time consuming benchmarks we usually scale them by a factor e.g. `n/10`. See benchmarks that do something like this and do the same here as well.

Reply:
Even for 100 elements, this takes like a few `ms`. Need to check what the problem is. Probably the use of `traverse`.

Reply:
Yes, I know that is perf issue. But irrespective of that we can use a scaling factor to reduce the timing appropriately, instead of using a constant value.